### DISCUSSION TITLE
Hide Items that don't belong to you in Ground Items plugin

### DISCUSSION THREAD

**Discussion by XITangoIX** [runelite/runelite#15003] [Ideas]
Ironmen can't pickup items that belong to other players so a toggle option to hide items they cant pickup would be a very nice QoL

It's also very convenient as items in your highlighted list that belong to other players will trigger notifications/lootbeams which is very annoying.

Edit: The "Only Show Loot" actually does this but only shows loot from npcs and yourself. It hides other things you can pick up like natural item spawns and ammo you fire. So the ideal solution is a toggle that only hides items you can't pick up.

**Comment by Nightfirecat** [comment]
This should already be covered by the "Show own loot" option in the ground items plugin.

**Reply by XITangoIX** [reply]
thanks, didnt realise that was there. However it also hides natural item spawns and ammo after you shoot it which isnt ideal. also i noticed that dropping highlighted items from your inventory doesnt trigger notifications while the only show loot is turned on.

still think it would be better for an alternative option to exist that allows the above while still hiding items from other players.

**Reply by Enriath** [reply]
To the client, there is no difference between an item you can pick up, and one that you can't; Once you try to pick it up, you're told by the server if you cannot. As a result, the plugin has to use other information to determine whether it's yours.
Global spawns aren't something known to the client, and would have to be somehow crowdsourced; If anyone's going to have a list, it'll be the wiki if they get around to crowdsourcing that.
For ammo, it could be made to be considered yours, but I imagine the implementation wouldn't be too pretty. It's also not a problem with the final tier of Ava's device, as no ammo will end up on the ground.

As for items you dropped not getting a notification... why would you want a notification for something you explicitly decided to drop? I'm struggling to see a use case given that the current notification system is to make AFKing nicer, as you can be alerted when something you're looking for is dropped.
